README.md

```diff
@@ -32,3 +54,38 @@ var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
 ```
+
+class Vector {
+  constructor(x,y){
+    this.x = x;
+    this.y = y;
+  };
+plus(vector){
+  let addedVector = new Vector(this.x + vector.x, this.y + vector.y)
+  return addedVector
+  };
+minus(vector){
+
+  let minusVector = new Vector(this.x - vector.x, this.y - vector.y)
+  console.log(minusVector)
+
+  };
+getLength(vector){
+  //(a^2 + b^2 = c^2)
+  //return Math.sqrt(Math.pow(sideA, 2) + Math.pow(sideB, 2));
+  let length = Math.sqrt(Math.pow(this.x,2) +Math.pow(this.y,2))
+    return length
+
+  };
+
+
+};
+
+
```

Comment:
Good job with the argument and variable names. The getLength function doesn't require an argument.